src/main/java/picard/fingerprint/CheckFingerprint.java

```diff
@@ -129,37 +134,65 @@ protected int doWork() {
             outputSummaryMetricsFile = new File(OUTPUT + FINGERPRINT_SUMMARY_FILE_SUFFIX);
         }
 
-        IOUtil.assertFileIsReadable(INPUT);
```

Comment:
I think I'll put it back even though it is checked in CustomCommandLineValidation - it's a pretty quick check.